nvflare/fuel/f3/drivers/aio_grpc_driver.py

```diff
@@ -257,7 +257,7 @@ def supported_transports() -> List[str]:
 
     @staticmethod
     def capabilities() -> Dict[str, Any]:
-        return {DriverCap.HEARTBEAT.value: True, DriverCap.SUPPORT_SSL.value: True}
+        return {DriverCap.HEARTBEAT.value: False, DriverCap.SUPPORT_SSL.value: True}
```

Comment:
Why is "DriverCap.HEARTBEAT.value: False"?
I thought GRPC will send heartbeat itself?

Reply:
When this is false, SFM will send its own heartbeat for the connection. By doing this, we have heartbeat in both layers. This makes it easier to control heartbeat interval across all drivers.

Reply:
Can we use another key/flag to determine whether to send SFM heartbeat or not?
Otherwise it is confusing to users as AIO GRPC driver DOES have the ability to send hearbeat.

Reply:
How about SEND_HEARTBEAT?

Reply:
That name should be fine.

We just need to make sure that this flag is not a driver capability.

As it is only related to SFM.